contracts/bridges/succinct/SuccinctBridgeReceiver.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.15;
+
+import "../BaseBridgeReceiver.sol";
+import "./ITelepathy.sol";
+
+contract SuccinctBridgeReceiver is BaseBridgeReceiver, ITelepathyHandler {
```

Comment:
Since this receiver is payable, it would be better to inherit `SweepableBridgeReceiver`, which inherits the `BaseBridgeReceiver`.

Reply:
Receiver shouldn't be payable, going to keep it as `BaseBridgeReceiver` for now.

Reply:
`SweepableBridgeReceiver` also allows the admin to sweep ERC20 assets mistakenly sent to this address, so still worth considering.

Reply:
I see, will add then.